packages/cli/package.json

```diff
@@ -89,11 +88,11 @@
     "@azure/arm-resources": "^4.0.0",
     "@azure/arm-sql": "^7.0.2",
     "@azure/arm-subscriptions": "^3.0.0",
-    "@azure/core-http": "^1.2.0",
-    "@azure/identity": "^1.3.0",
-    "@azure/ms-rest-nodeauth": "^3.0.6",
-    "@microsoft/teamsfx-api": "^0.1.0-rc.4",
-    "@microsoft/teamsfx-core": "^0.1.0-rc.4",
+    "@azure/core-http": "^1.2.2",
+    "@azure/identity": "^1.2.3",
```

Comment:
I am ok with upgrade to newer version for dependencies. But a little bit concern about downgrade the dependencies. dev probably upgrades to newer version for a reason.